client/python/requirements.txt

```diff
@@ -7,7 +7,7 @@ certifi==2022.12.7
 chardet==4.0.0
 colorama==0.4.4
 docutils==0.16
-idna==3.1
+idna==2.5
```

Comment:
```
ERROR: Cannot install -r client/python/requirements.txt (line 32) and idna==3.1 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested idna==3.1
    requests 2.25.1 depends on idna<3 and >=2.5

```